src/main/java/lotto/domain/LottoAutoMaker.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoAutoMaker {
+
+
+    private static final List<LottoNumber> numbers = IntStream.range(LottoNumber.MIN_VALUE, LottoNumber.MAX_VALUE + 1)
```

Comment:
로또 후보 번호 정적 상수 선언 좋습니다 👍 다만 대문자로 표기해주시면 더 좋을 것 같네요 😄 
```suggestion
    private static final List<LottoNumber> CANDIDATE_NUMBERS = IntStream.range(LottoNumber.MIN_VALUE, LottoNumber.MAX_VALUE + 1)
```